src/answers-headless.ts

```diff
@@ -48,6 +48,10 @@ export default class AnswersHeadless {
     this.stateManager.dispatchEvent('query/setSource', source);
   }
 
+  setSearchAggregationId(uuid: string): void {
```

Comment:
should we just call it `setAutocompleteSessionId`? or did Rose have other thoughts

Reply:
nope, it just thought it would be a bit clearer in what autocompleteSessionId is doing. But I can just change it, not a problem